app/components/post_draft/send_action/index.js

```diff
@@ -20,7 +20,10 @@ function SendButton(props) {
 
     if (props.disabled) {
         return (
-            <View style={style.sendButtonContainer}>
+            <View
+                testID='disabled_send_button'
```

Comment:
this is great, but don't you  think it would be better if we add some sort of prefix patterns for the testID's ?

Reply:
Thanks and yeah, I think it's a good point but can you give a concrete example?
The use case came to my mind is adding prefix or suffix to make the testID unique. It may be applicable to other components but in this `disabled_send_button` example, not much use for it. It's used for channel post list and thread but when I inspected, it's not rendered at the same time. For suffix, we might going to add dynamic ID, like post ID, when assigning testID to elements that comprises `Post` component (like for user profile).

I'll include in the docs whatever is agreed for testID naming convention.

Reply:
not for this PR, but maybe this can be `<this component>.<child>.<action or identifier>` or something like that 0/5 on the format / prefix / suffix but 5/5 that we need it to avoid repetition of testID's